sympy/simplify/tests/test_simplify.py

```diff
@@ -720,3 +720,29 @@ def test_clear_coefficients():
     assert clear_coefficients(S.Infinity, x) == (S.Infinity, x)
     assert clear_coefficients(-S.Pi, x) == (S.Pi, -x)
     assert clear_coefficients(2 - S.Pi/3, x) == (pi, -3*x + 6)
+
+def test_nc_simplify():
+    from sympy.simplify.simplify import nc_simplify
+    a, b, c = symbols('a b c', commutative = False)
+    x = Symbol('x')
+
+    def _check(expr, simplified, deep=True):
+        assert nc_simplify(expr, deep=deep) == simplified
```

Comment:
At first I was, but then `expand((c*a)**-1) == (c*a*)**-1` and not `expand((c*a)**-1 == a**-1*c**-1` so I'd have to add a keyword argument to ignore the `expand` check for some of the tests. I didn't in the end because all of the expressions I used are sufficiently short that I know with certainty the simplified version is correct. But I guess I could add an `ignore` keyword argument, and have the check anyway just in case, if you think it's worth it. Or, alternatively, `expand((c*a)**-1` should probably be `a**-1*c**-1` - I think I could write the PR for it today already, and then if it's merged first, I wouldn't need an additional keyword argument.

Reply:
Wait, I don't even need a separate PR, I can do it right here.

Reply:
There is a test in `core/tests/test_expand.py` that does `expand(A*B*(A*B)**-1) == A*B*(A*B)**-1` for non-commutative `A, B` after [this issue](https://github.com/sympy/sympy/issues/6558). So I'm somewhat unsure if that was intentional. After the change I'm proposing to do it'll be `expand(A*B*(A*B)**-1) == 1`.